docs/configure_android_sdk.md

```diff
@@ -0,0 +1,179 @@
+#Configure Android SDK
```

Comment:
should this doc have a different name ? Right now it is named `configure_android_sdk` but it is mostly related with RUM configuration.

Reply:
The way I see it this page has a bit of mixed information about all the features, I don't really see what is the real intention. Do you want to have here only the basic information about how to initialize the SDK by passing the credentials and a minimum configuration and then move all the RUM/Logs/Apm related information in different pages ?